contracts/Identity.sol

```diff
@@ -3,31 +3,63 @@ pragma solidity ^0.4.24;
 import "zos-lib/contracts/Initializable.sol";
 import "contracts/KeyManager.sol";
 import "openzeppelin-eth/contracts/cryptography/ECDSA.sol";
-import "openzeppelin-eth/contracts/ownership/Ownable.sol";
 
 
-contract Identity is Ownable, KeyManager {
+contract Identity is KeyManager {
   using ECDSA for bytes32;
 
-  constructor() public{
-    Ownable.initialize(msg.sender);
+  constructor(address owner) public {
+    // Add MANAGEMENT_KEY
+    bytes32 key = addressToKey(owner);
+    keys[key].purposes.push(1);
+    keys[key].keyType = 1;
+    keysByPurpose[1].push(key);
   }
 
-  // @param _toSign Hash to be signed. Must be generated with abi.encodePacked(arg1, arg2, arg3)
-  // @param _signature Signed data
-  function isSignatureValid(
+
+  /**
+   * @dev Proxy execution
+   * @param _to address smart contract to call
+   * @param _value uint256 wei supply for proxy execution
+   * @param _data bytes ABI encoded call data
+   */
+  function execute(
```

Comment:
I can not think of any valid use case for delegateCall at the moment. You can call the contract directly. You delegate call only in a approve flow as far as I can tell. I am missing something?

Reply:
Ok, you can do that but what about external 725 identities. Delegation/external key manager is not a requirement for us as our use case is covered by call. There is still a debate around this and it is not part of the current 725 standard and I think we should spend to much energy on this. I think using external keymanagers is not a requirement for identites as an identity can register to key managers.

Considering that the user owns his identity, upgradable keymanager should be upgraded only by the user.

Reply:
External key manager was maybe the wrong wording. It should still be owned by the user.

Anyway, I agree. Let's not spend to much time on that. It would only be extra functionality we currently don't need. 👍